### PR TITLE
Feature/steam integration

### PR DESCRIPTION
**Pull request by ambid17**
the editor code is WIP, but the steam integration is done

### CODE REVIEW

File: BunnyHopMaster/Assets/Scripts/SceneLoader.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    void GetRemoteAssetBundle()
+    {
+        StartCoroutine(DownloadAssetBundle());
+    }
+
+    IEnumerator DownloadAssetBundle()
+    {
+        WWW www = new WWW(assetURL);
```
Comment (0xLeif) [inline BunnyHopMaster/Assets/Scripts/SceneLoader.cs:31]: Do you never use this `www` var?

File: BunnyHopMaster/Assets/Scripts/Singletons/GameManager.cs [hunk 2]
```diff
@@ -30,6 +32,22 @@ void Awake()
...
+        StartSteam();
+    }
+
+    private void StartSteam()
+    {
+        try
+        {
+            if (!SteamClient.IsValid)
+            {
+                SteamClient.Init(AppId, true);
```
Comment (0xLeif) [inline BunnyHopMaster/Assets/Scripts/Singletons/GameManager.cs:44]: 🥳 🎊

**Review by 0xLeif — APPROVED** [review]

**Comment by 0xLeif** [comment]
@ambid17 Is there any way to get debug logs from people that play through steam?

File: BunnyHopMaster/Assets/Scripts/SceneLoader.cs (hunk 1 above)
Comment (ambid17) [reply BunnyHopMaster/Assets/Scripts/SceneLoader.cs:31]: That’s the WIP code :D

File: BunnyHopMaster/Assets/Scripts/Singletons/GameManager.cs (hunk 2 above)
Comment (ambid17) [reply BunnyHopMaster/Assets/Scripts/Singletons/GameManager.cs:44]: 🔥

**Comment by ambid17** [comment]
That’s a good idea, they have good logging. I’ll add a ticket!

**Pull request merged**